### ISSUE TITLE
Confusion on Question #11

### ISSUE THREAD

**Issue by thapashekhar** [leombastos/2023_AdvPA#78]
The question is What was the proportion of high and low yield areas for each of the zones.  Is it asking the proportion of high yield zone area and proportion of low yield zone area or proportion of yield classes in each cluster?

**Comment by saramctabi** [comment]
I though it should be about the yield classes, but now I doubt it.

**Comment by thapashekhar** [comment]
Oh yes, It should be the proportion of yield classes in each zone. Thanks @saramctabi for the confirmation.

**Comment by AmrinderJakhar98** [comment]
Me to got now confused the cluster after smoothing and yielding classes are not the same??? @saramctabi @thapashekhar

**Comment by thapashekhar** [comment]
Yes, you are right, @AmrinderJakhar98.
Based on % of median value of yield, I have 3 yield classes: high yield, medium yield and low yield. For each cluster (I have 2 clusters), we can determine number of data points falling under these yield classes. For example, in cluster 1, let's say, there are 25 high yield data points out of 100 high yield data points in the entire field. The proportion of high-yield areas in cluster 1 will be 25/100 = 25%. (we have code chunk for this algorithm)
This is how I addressed the question 11.

**Comment by AmrinderJakhar98** [comment]
"Yes, I did it in a similar way as @Leoanish did. I don't think you need to create yield classes, but I could be wrong. @thapashekhar, I also started with creating three classes like you did, but as I progressed towards the end, I didn't think we needed them. It's confusing to me, especially since the workflow mentions performing yield stability analysis. Can anybody else share their point of view?"

**Comment by thapashekhar** [comment]
@Leoanish, Based on one-year yield data, we can classify the field into high, medium, and low yielding areas. The only we cannot say is how consistent those yielding areas are.
The classification is based on yield class, not stclass (yield stability- class).

**Comment by thapashekhar** [comment]
@leombastos, could you please give us a hint on this?

**Comment by Leoanish** [comment]
Yeah but how can we can the pixels are high, medium, or low yielding in nature? I think we did it after standardizing the yield data and then used the cv and sd classification to go through that process.

But with management zones, we clustered the data points and k mean was the main reference for us to differentiate the 2 different zones

**Comment by thapashekhar** [comment]
we can categorize the pixels based on standardized value. For example if pixel yield value is less than 80 % of the median yield value, we can categorize that pixel as low yield pixel. Here we are using only standardized value to categorize the yield class, we are not doing stability classification.
 
But in lecture class, we categorized into yield classes by mean standardized yield (mean_pixels) and stability classes by CV.
